sos/report/plugins/networking.py

```diff
@@ -82,7 +82,10 @@ def setup(self):
 
         self.add_cmd_output("ip -o addr", root_symlink="ip_addr",
                             tags='ip_addr')
-        self.add_cmd_output("route -n", root_symlink="route", tags='route')
+        self.add_cmd_output("route -n -4", root_symlink="route",
+                            tags='route')
+        self.add_cmd_output("route -n -6", root_symlink="route_v6",
+                            tags='route_v6')
```

Comment:
Hmm, not sure about dropping another root symlink with this. If anything, I've been mulling over reducing the number of root symlinks we create.

@pmoravec your thoughts?

Reply:
If we don't want to create root symlink, I'm not sure there is point at all in pushing this PR. `route` is deprecated for quite some time now in favour of `ip route` and we run somewhere else `ip route show table all` already - so we do have the required data in the sosreport, it's just that the structure is inconsistent at the moment.

If it anyhow happened that someone runs sosreport on a system with `route` but without `ip route` I am not sure this system is recent enough to be even plugged into IPv6/dual-stack networking, so we may be also doing useless thing...

Reply:
The root symlinks were traditionally for the most referenced parts of sos collections. Over time, as our collections have grown in both scope and complexity, we've shied away from dropping more symlinks in the archive root, in an effort to keep things consumable at a glance.

Removal of root symlinks is something we tend to avoid mainly because of just how long it has been "common" knowledge that you can untar an sos report, cd into the directory and immediately get certain networking information or packaging information without having to hunt it down in the subdirectories.

At the same time, while not necessarily in conjunction, we've added better accounting for the collections in a report and through `sos_reports/manifest.json` we provide an easy way for automation to pick out specific collections via tags so that those systems aren't reliant on keeping up with the exact command construction for every version of sos they might consume reports from.

If `route` is deprecated, I'd pause before adding a new collection for it just for the sake of having a new root symlink added. Instead, I'd look at the applicability of maintaining those collections and symlinks going forward. If `ip route` is the "correct" way to get this information, what do you think about dropping the current `route` collection entirely? Is `ip route show table all` sufficient to cover all the same information? We could perhaps drop the `route` collection and replace the symlink with `ip_route` pointing to this command, which seems to include IPv6 information as well, if there is significant concern over removal of the root symlink.

Reply:
I think it's all about compatibility now... The output format of `route` and `ip route` is entirely different, so back to your initial point -- if there is someone automating sosreport's output and parsing the output, we would brake them.

Knowledge-wise, we are not losing anything by dropping the current `route` collection. Supportability-wise, we may break some people (even though they should follow the deprecation notice from years ago and adjust). I'm not sure there is a silver bullet here. For consistency we should either extend what we collect via `route` or remove it completely. My personal view would be to drop `route` because it's outdated anyway, but again, it's all about backwards-compatibility.

For reference, https://wiki.linuxfoundation.org/networking/net-tools

Reply:
Having one `ip_route` root symlink and dropping `route ..` commands entirely makes the most sense to me - until some support or bigger group of users rely on it (though even then I would have this as a goal and warn those users to prepare for the change, in this case).